packages/belos/src/BelosGmresPolyOp.hpp

```diff
@@ -709,10 +709,14 @@ namespace Belos {
       std::cout << "GEEV solve : info = " << info << std::endl;
     }
 
-    // Set index for sort function and sort Harmonic Ritz Values:
+    // Set index for sort function, verify roots are non-zero,
+    // and sort Harmonic Ritz Values:
+    const MagnitudeType tol = (MagnitudeType) 10 * Teuchos::ScalarTraits<ScalarType>::eps();
```

Comment:
Why not just `Teuchos::ScalarTraits<MagnitudeType>::eps()`?

Reply:
Because that's how it is in the BelosMVOPTester.hpp I copied from?  :P  I dunno.  I'll go change it.